services/tracing/migrations/0230821113939_initial_tracing.up.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE steps (
```

Comment:
Try  and wrap your migrations in a transaction (BEGIN; ....; COMMIT;) 

This makes sure that when a migration fails, it is not partially applied

Reply:
fixed in #30